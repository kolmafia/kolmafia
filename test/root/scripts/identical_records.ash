record rec1
{
    int x;
    int y;
};

record rec2
{
    int x;
    int y;
};

rec1 var1 = new rec1(10, 20);
rec1 var2 = new rec2(30, 40);

print("rec1 -> rec1: x = " + var1.x + " y = " + var1.y);
print("rec2 -> rec1: x = " + var2.x + " y = " + var2.y);
