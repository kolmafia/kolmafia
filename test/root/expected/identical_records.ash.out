rec1 -> rec1: x = 10 y = 20
rec2 -> rec1: x = 30 y = 40
